python-package/mlbox/preprocessing/reader.py

```diff
@@ -3,99 +3,105 @@
 # Author: Axel ARONIO DE ROMBLAY <axelderomblay@gmail.com>
 # License: BSD 3 clause
 
-import numpy as np
-import pandas as pd
 import pickle
-from sklearn.preprocessing import LabelEncoder
 import os
-import warnings
 import time
+import numpy as np
+import pd as pd
+import ipyparallel as ipp
+from sklearn.preprocessing import LabelEncoder
 
 os.system("ipcluster start --profile=home &")
-import ipyparallel as ipp
 
```

Comment:
@AxeldeRomblay 

Was it done on purpose to import `ipyparallel ` after doing `os.system("ipcluster start --profile=home &")` ?